Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -266,7 +266,8 @@ int vtkMRMLMarkupsNode::EndModify(int previousDisableModifiedEventState)
     || this->GetCustomModifiedEventPending(vtkMRMLMarkupsNode::PointPositionDefinedEvent) > 0
     || this->GetCustomModifiedEventPending(vtkMRMLMarkupsNode::PointPositionUndefinedEvent) > 0
     || this->GetCustomModifiedEventPending(vtkMRMLMarkupsNode::PointPositionMissingEvent) > 0
-    || this->GetCustomModifiedEventPending(vtkMRMLMarkupsNode::PointPositionNonMissingEvent) > 0);
+    || this->GetCustomModifiedEventPending(vtkMRMLMarkupsNode::PointPositionNonMissingEvent) > 0
+    || this->GetCustomModifiedEventPending(vtkMRMLMarkupsNode::PointAboutToBeRemovedEvent) > 0);
```

Comment:
PointAboutToBeRemovedEvent should not be added here, because it does not indicate an actual modification yet, so there is no need to make any updates yet. To make this clear (and prevent someone from thinking that the event was unintentionally missed from here) please add a comment here that PointAboutToBeRemovedEvent should not be added here.